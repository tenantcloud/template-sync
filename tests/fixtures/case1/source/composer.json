{
	"name": "package-name",
	"description": "description",
	"require": {
		"r1": "^1.0",
		"r3": "^1.0"
	},
	"require-dev": {},
	"scripts": {
		"cs-fix": "php-cs-fixer",
		"test": "phpunit",
		"extra": "extra"
	}
}
