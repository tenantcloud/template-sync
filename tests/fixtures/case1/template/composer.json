{
	"name": "some-stub-name",
	"description": "some-stub-description",
	"require": {
		"r1": "^2.0",
		"r2": "^3.0",
		"package-name": "dev-master"
	},
	"require-dev": {
		"to-override": "^1.0"
	},
	"scripts": {
		"cs-fix": "./php-cs-fixer",
		"test": "./phpunit"
	}
}
