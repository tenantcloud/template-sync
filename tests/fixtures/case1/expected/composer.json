{"name":"package-name","description":"description","require":{"r1":"^2.0","r3":"^1.0","r2":"^3.0"},"require-dev":{},"scripts":{"cs-fix":"./php-cs-fixer","test":"./phpunit","extra":"extra"}}
